### DISCUSSION TITLE
Why is ImageMagick creating multi-page documents?

### DISCUSSION THREAD

**Discussion by tomsutherland** [ImageMagick/ImageMagick#4125] [Help]
I am using ImageMagick to convert a file to various different file types.

I take a PSD and run the following for example:

magick inputfile.psd outputfile.pdf

The PDF that comes out has two identical pages and I've no idea why. It should only have one. Any ideas?

Thanks Tom

**Comment by snibgo** [comment]
I suppose your input has two images. What does "magick inptfile.psd info:" say? If you want just the first image, then:
```
magick inputfile.psd[0] outputfile.pdf
```
